### PR TITLE
HDDS-5038. Create and update ZH translation of SCM-HA.md and OM-HA.md in doc

### PR DESCRIPTION
**Pull request by sky76093016**
## What changes were proposed in this pull request?

Increase translation documents for SCM-HA and OM-HA.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5038

## How was this patch tested?

Local test.

### CODE REVIEW

File: hadoop-hdds/docs/content/feature/OM-HA.zh.md [hunk 1]
```diff
@@ -24,12 +24,13 @@ summary: Ozone 用于避免单点故障的高可用设置
...
 
-Ozone 有两个leader节点（用于键管理的 *Ozone Manager* 和用于块空间管理的 *Storage Container Management* ）以及存储节点（数据节点）。数据是借助 RAFT 共识算法在数据节点之间复制。
+Ozone 有两个元数据管理节点（用于键管理的 *Ozone Manager* 和用于块空间管理的 *Storage Container Management* ）以及多个存储节点（数据节点）。数据是借助 RAFT 共识算法在数据节点之间复制。
 
-为了避免任何单点故障，leader 节点也应该具备高可用的设置。
+为了避免任何单点故障，元数据管理节点也应该具备高可用的设置。
 
-1. Ozone Manager 的高可用是借助 RAFT 协议实现的；
-2. Storage Container Manager 的高可用则 [还在实现中]({{< ref path="scmha.md" lang="en">}}) 。
+Ozone 管理器和 Storage Container Manager 都支持 HA。在这种模式下，内部状态通过 RAFT (使用 Apache Ratis )复制。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/OM-HA.zh.md:31]: Ozone 管理器=> Ozone Manager

File: hadoop-hdds/docs/content/feature/OM-HA.zh.md [hunk 2]
```diff
@@ -24,12 +24,13 @@ summary: Ozone 用于避免单点故障的高可用设置
...
+
+本文档解释了 Ozone 管理器(OM) HA的高可用设置，请查看[本页]({{< ref "SCM-HA" >}})的 SCM HA。虽然它们可以独立地为 HA 进行设置，但可靠的、完全的 HA 设置需要为两个服务启用 HA。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/OM-HA.zh.md:33]: Can you double check the link to SCM HA in the rendered page?

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+---
+title: "SCM 高可用"
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.zh.md:2]: Suggested title: 高可用SCM

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+<div class="alert alert-warning" role="alert">
+请注意，SCM-HA 尚未准备好在安全环境中进行生产。 安全工作正在进行中，将很快完成。
+</div>
+
+为了避免任何单点故障，元数据管理器节点还应该具有 HA 设置。
+
+Ozone Manager 和 Storage Container Manager 都支持 HA。在这种模式下，内部状态通过 RAFT (使用 Apache Ratis )复制。
+
+本文档解释了 Storage Container Manager (SCM)的 HA 设置，请在[本页]({{< ref "OM-HA" >}})中查看 Ozone Manager (OM)的 HA 设置。虽然它们可以独立地为 HA 进行设置，但可靠的、完全的 HA 设置需要为两个服务启用 HA。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.zh.md:36]: OM-HA is on a different page， can  check the link？

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+定义的前缀可以用来定义每个 SCM 服务的地址:
+
+```XML
+<property>
+   <name>ozone.scm.address.cluster1.scm1</name>
+   <value>host1</value>
+</property>
+<property>
+   <name>ozone.scm.address.cluster1.scm1</name>
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.zh.md:78]: scm1=》scm2

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+   <value>host2</value>
+</property>
+<property>
+   <name>ozone.scm.address.cluster1.scm1</name>
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.zh.md:82]: scm1=》scm3

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+根据 `ozone.scm.primordial.node.id`，初始化进程将在第二个/第三个节点上被忽略，引导进程将在除原始节点外的所有节点上被忽略。
+
+## 实现细节
+
+SCM HA 使用 Apache Ratis 在 SCM HA 仲裁的成员之间复制状态。每个节点在本地 RocksDB 中维护块管理元数据。
+
+这个复制过程是 OM HA 复制过程的一个简单版本，因为它不使用任何双缓冲区(SCM 请求的总体 db 吞吐量更低)。
+
+数据节点将所有报告(容器报告、管道报告……)并发发送给 *所有* 数据节点。只有 leader 节点可以分配/创建新的容器，并且只有 leader 节点可以将命令返回给数据节点。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.zh.md:120]: *所有* 数据节点， this does not seem right？should be 所以的SCM 节点

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+// use debug ldb to check scm db on all the machines
+bin/ozone debug ldb --db=/tmp/metadata/scm.db/ ls
+ 
+ 
+bin/ozone debug ldb --db=/tmp/metadata/scm.db/ scan --with-keys --column_family=containers
+```
+
+## 从现有的SCM迁移
+
+可以在任何 Ozone 集群上打开 SCM HA。 首先启用 Ratis（`ozone.scm.ratis.enable`）并为 Ratis ring 配置一个节点（`ozone.scm.nodes.NAME` 应该有一个元素）。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.zh.md:158]: NAME should be serviceId

**Review by xiaoyuyao — COMMENTED** [review]
Thanks @sky76093016  for the patch. It LGTM overall, a few comments added inline.

File: hadoop-hdds/docs/content/feature/OM-HA.zh.md (hunk 1 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/OM-HA.zh.md:31]: Solve it.

File: hadoop-hdds/docs/content/feature/OM-HA.zh.md (hunk 2 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/OM-HA.zh.md:33]: image
Click this button to jump to this page, is it correct?

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md (hunk 3 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:2]: Solve it.
Does the title of OM-HA.zh.md need to be changed to ‘高可用OM’?

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone包含两个元数据管理节点(用于键管理的 *Ozone Manager* 和用于块空间管理的 *Storage Container management* )和多个存储节点(数据节点)。通过 RAFT 共识算法实现数据在数据节点之间的复制。
+
+<div class="alert alert-warning" role="alert">
+请注意，SCM-HA 尚未准备好在安全环境中进行生产。 安全工作正在进行中，将很快完成。
```
[the remark this answers is not in the document]
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:29]: Solve it.

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md (hunk 4 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:36]: image
Click this button to jump to this page, is it correct?

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md (hunk 5 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:78]: Solve it.

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md (hunk 6 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:82]: Solve it.

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md (hunk 7 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:120]: image
So should we modify the original text synchronously?

File: hadoop-hdds/docs/content/feature/SCM-HA.zh.md (hunk 8 above)
Comment (sky76093016) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:158]: image
Solve it.
But should we modify the original text synchronously?

Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/feature/SCM-HA.zh.md:158]: Yes, I think the original EN doc should be updated as well.

**Review by xiaoyuyao — APPROVED** [review]
Thanks @sky76093016 for the contribution. LGTM, +1.

**Pull request merged**